### PR TITLE
Subscriptions v3 spectral errors corrections

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
Missing descriptions will be added in future PRs, along with Permissions.

#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

### CODE REVIEW

File: VTEX - Subscriptions API (v3).json [hunk 1]
```diff
@@ -1727,252 +1603,346 @@
...
                     "status": {
                         "type": "string",
-                        "nullable": true
+                        "description": "Subscription cycle execution status. The possible values are:\r\n- `TRIGGERED`: Execution has been triggered. \r\n- `IN_PROCESS`: Execution is being processed by the system.\r\n- `FAILURE`: An internal error occurred during the subscription execution.\r\n- `SUCCESS`: Successful cycle processing.\r\n- `EXPIRED`: The subscription was not renewed, and the deadline for which it was valid has passed.\r\n- `ORDER_ERROR`: Cycle was not executed due to an error in order placement.\r\n- `PAYMENT_ERROR`: Cycle was not executed due to an error in the payment.\r\n- `SKIPED`: A subscription Cycle execution was skipped, and the subscription will be executed in the next cycle.\r\n- `SUCCESS_WITH_NO_ORDER`: Cycle was executed successfully, and the linked order has no items.\r\n- `SUCCESS_WITH_PARTIAL_ORDER`: Cycle was executed successfully, and has a linked partial order.\r\n- `RE_TRIGGERED`: Re-execution was triggered manually.\r\n- `SCHEDULE_UPDATED`: Date of the next subscription cycle execution has been updated.",
```
Comment (julia-rabello) [inline VTEX - Subscriptions API (v3).json:1699]:
```suggestion
                        "description": "Subscription cycle execution status. The possible values are:\r\n- `TRIGGERED`: Execution has been triggered. \r\n- `IN_PROCESS`: Execution is being processed by the system.\r\n- `FAILURE`: An internal error occurred during the subscription execution.\r\n- `SUCCESS`: Successful cycle processing.\r\n- `EXPIRED`: The subscription was not renewed, and the period for which it was valid has ended.\r\n- `ORDER_ERROR`: Cycle was not executed due to an error in order placement.\r\n- `PAYMENT_ERROR`: Cycle was not executed due to an error in the payment.\r\n- `SKIPED`: A subscription Cycle execution was skipped, and the subscription will be executed in the next cycle.\r\n- `SUCCESS_WITH_NO_ORDER`: Cycle was executed successfully, and the linked order has no items.\r\n- `SUCCESS_WITH_PARTIAL_ORDER`: Cycle was executed successfully, and has a linked partial order.\r\n- `RE_TRIGGERED`: Execution retry was triggered manually.\r\n- `SCHEDULE_UPDATED`: Date of the next subscription cycle execution has been updated.",
```

File: VTEX - Subscriptions API (v3).json [hunk 2]
```diff
@@ -1727,252 +1603,346 @@
...
+                    },
+                    "customerId": {
+                        "type": "string",
+                        "description": "Customer ID."
                     },
                     "date": {
                         "type": "string",
-                        "format": "date-time"
+                        "format": "date-time",
+                        "description": ""
```
Comment (julia-rabello) [inline VTEX - Subscriptions API (v3).json:1726]: add description

File: VTEX - Subscriptions API (v3).json [hunk 3]
```diff
@@ -2247,13 +2141,85 @@
                         "nullable": true
```
Comment (julia-rabello) [inline VTEX - Subscriptions API (v3).json:2141]: add description for this property and others in this schema

File: VTEX - Subscriptions API (v3).json [hunk 4]
```diff
@@ -2247,13 +2141,85 @@
...
+                            "interval": {
+                                "type": "integer",
+                                "format": "int32"
+                            }
+                        },
+                        "additionalProperties": false
                     },
                     "validity": {
-                        "$ref": "#/components/schemas/ValidityResponse"
+                        "type": "object",
```
Comment (julia-rabello) [inline VTEX - Subscriptions API (v3).json:2162]: add description

File: VTEX - Subscriptions API (v3).json [hunk 5]
```diff
@@ -2247,13 +2141,85 @@
...
+                                "format": "date-time",
+                                "nullable": true
+                            }
+                        }
                     },
                     "purchaseDay": {
-                        "$ref": "#/components/schemas/PurchaseDay"
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "",
```
Comment (julia-rabello) [inline VTEX - Subscriptions API (v3).json:2181]: add description

File: VTEX - Subscriptions API (v3).json (hunk 4 above)
Comment (Isabella-Veloso) [reply VTEX - Subscriptions API (v3).json:2162]: There were too many small components schemas with only one or two mentions. The whole `PlanResponse` schema has to be checked according to the API, to see if it is up to date. I will do that on the next PRs, in this one I was organizing the schema that was part of another endpoint, just had to add a part in there, so I will ask you to approve the PR with this exception. I will add the `PlanResponse` descriptions in the next PRs.

File: VTEX - Subscriptions API (v3).json (hunk 3 above)
Comment (Isabella-Veloso) [reply VTEX - Subscriptions API (v3).json:2141]: There were too many small components schemas with only one or two mentions. The whole `PlanResponse` schema has to be checked according to the API, to see if it is up to date. I will do that on the next PRs, in this one I was organizing the schema that was part of another endpoint, just had to add a part in there, so I will ask you to approve the PR with this exception. I will add the `PlanResponse` descriptions in the next PRs.

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**